plugin_manager.py

```diff
@@ -47,7 +46,20 @@
 PLUGIN_DIRECTORY = _env["python_directory_user"]
 loop = babase._asyncio._asyncio_event_loop
 
+open_popups = []
```

Comment:
Is it just to keep hard reference of class instance?
Do the reference cause issue for popups?
And cant we just `del(self)` in `def _back():` or something?
@vishal332008

Reply:
I was planning on future purposes like making the popups appear back again, like whenever you change ui scales, the whole window refreshes, and whole plugin list refreshes and stuff, just eh, so i was thinking, in future, maybe to set up a little smth to load faster and load back the popups quickly, so i saved them a bit, but for erasing now, i did that i made now, am i making sense or am i yapping random bs?

Reply:
> I was planning on future purposes like making the popups appear back again, like whenever you change ui scales, the whole window refreshes, and whole plugin list refreshes and stuff, just eh, so i was thinking, in future, maybe to set up a little smth to load faster and load back the popups quickly, so i saved them a bit, but for erasing now, i did that i made now, am i making sense or am i yapping random bs?

what i understand from that is that u added that for doing something with it in the future

Reply:
> > Cant we just `del(self)` in `def _back():` or something?
> 
> but if this works then it would be a better solution

Bro, i seriously can't understand, the transition outscale kinda already deletes it, i just changed a bit so that it'll delete it from the array, idk what exactly you're saying

Reply:
> > > Cant we just `del(self)` in `def _back():` or something?
> > 
> > 
> > but if this works then it would be a better solution
> 
> Bro, i seriously can't understand, the transition outscale kinda already deletes it, i just changed a bit so that it'll delete it from the array, idk what exactly you're saying

so why did you make the list then?

Reply:
hello?

Reply:
alright understood